setup.py

```diff
@@ -91,11 +73,11 @@ def run_tests(self):
         'keyring',
         'matplotlib',
         'notebook',
-        'numpy>=1.12',
+        'numpy <1.16, >=1.12',
```

Comment:
Why < 1.16?

Reply:
Needed until the following bug is resolved:

https://github.com/PyCQA/pylint/issues/2721

Reply:
The alternative, which I am certainly open to, is disabling those error messages from pylint. I figure we can resort to that if the bug takes a long time to resolve.